index.bs

```diff
@@ -641,7 +641,92 @@ To <dfn>obtain a report</dfn> given an [=attribution source=] |source| and an [=
     :: |source|'s [=attribution source/source identifier=].
 1. Return |report|.
 
-# Security consideration # {#security-considerations}
+# Report delivery # {#report-delivery}
+
+The user agent MUST periodically gather [=attribution reports=] from its [=attribution report cache=] and run [=queue a report for delivery=] on each report.
+
+To <dfn>queue a report for delivery</dfn> given an [=attribution report=] |report|, run the following steps:
```

Comment:
```suggestion
To <dfn>queue a report for delivery</dfn> given an [=attribution report=] |report|, run the following steps [=in parallel=]:
```

Since there are several "wait"s that could reorder reports.